spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -1511,8 +1523,6 @@ The following order, with the lower number having higher precedence, is used whe
 1. If the method is a Java default method, then its provided implementation is used.
 2. If the method has a _Resource Accessor Method_ return type, then the method is implemented as a _Resource Accessor Method_.
 3. If the method is annotated with `@Query`, then the method is implemented to run the corresponding Query Language query.
-4. If the method is annotated with a _Life Cycle_ annotation that defines the type of operation (`@Insert`, `@Update`, `@Save`, or `@Delete`), then the annotation determines how the method is implemented.
+4. If the method is annotated with a _Life Cycle_ annotation that defines the type of operation (`@Insert`, `@Update`, `@Save`, `@Delete`, or `@Find`), then the annotation determines how the method is implemented.
```

Comment:
After re-reading the section on life cycle methods, I would not consider `Find` to be categorized under life cycle methods.  The second sentence of the introductory paragraph on life cycle methods very clearly states, "Lifecycle methods allow the program to make changes to persistent data in the data store."  So `Find` does not fit under that.

```suggestion
4. If the method is annotated with `@Find` or a _Life Cycle_ annotation that defines the type of operation (`@Insert`, `@Update`, `@Save`, `@Delete`), then the annotation determines how the method is implemented.
```